packages/desktop/views/dashboard/wallet/tab-section/portfolio/components/TokenListRow.svelte

```diff
@@ -71,9 +73,11 @@
             </Text>
         </div>
     </div>
-    <div class="flex flex-row gap-2 text-start items-center">
-        <NetworkAvatar size="xs" networkId={token.networkId} />
-        <Text>{getNameFromNetworkId(token.networkId)}</Text>
+    <div class="h-full flex flex-row gap-2 justify-center items-center">
+        <div bind:this={anchor}>
```

Comment:
network badge is an avatar with a tooltip, we can use this here if we update the sizes

Reply:
we would need to add props to that to disable the border color/width and the absolute placement.
I think the best would be to add the tooltip to the networkAvatar component itself